### DISCUSSION TITLE
Make the Playground able to convert from JSON to DSL

### DISCUSSION THREAD

**Discussion by campidelli** [openfga/community#125] [Ideas]
Hi everyone!

This is more like a feature request instead of an issue. I am assuming the Playground utilizes the syntax-transformer under the hood to allow the transformation from DSL to JSON. It would be great to have the opposite as well, as we could choose to store either the DSL or the JSON file.

I also understand that the Playground isn't open source, is there any plan for it to be?

Cheers,,

Bruno

**Comment by rhamzeh** [comment]
Thanks for the suggestion @campidelli! We transferred it to an idea to keep the openfga/openfga issues focused on issues related to the Core API.

Just to make sure we tackle your needs, can you elaborate more on how you would be using it? As in what would your flow be?

**Reply by campidelli** [reply]
Sure!

I want to save my authorisation JSON file in a GitHub repository. If I want to change it, I want to go to the playground, convert it to DSL and make the necessary changes. Then I can parse it back to JSON and save it to my GitHub repo.

**Comment by aaguiarz** [comment]
Hi @campidelli 

You can now use the [CLI](https://github.com/openfga/cli?tab=readme-ov-file#transform-an-authorization-model) to achieve that, which we believe is a better approach.
